JSON/Mystery Widget.json

```diff
@@ -0,0 +1,22 @@
+{
+  "Author": "BakersDozenBagels",
+  "Compatibility": "Compatible",
+  "DefuserDifficulty": "Easy",
+  "Description": "Hides a widget until certain modules are solved. Tags: Push, Hidden-Widget",
+  "ExpertDifficulty": "VeryEasy",
+  "Ignore": [ "Mystery Widget, Cookie Jars, Divided Squares, Encrypted Hangman, Encryption Bingo, Four-Card Monte, Hogwarts, The Heart, The Swan, Button Messer, Random Access Memory, Turn The Keys, Tech Support, +FullBoss, +SemiBoss" ],
```

Comment:
I'm no maintainer, but this ignore list is written wrong

Reply:
I mean yeah its weird but idk tbh.

Reply:
This is what I got form the JSON generator. I guess I needed to add quotes in there. Since I can't edit this, I would appreciate someone else doing that.

Reply:
Also can you add the source line:
"SourceUrl": "https://github.com/BakersDozenBagels/KTaNEMysteryWidget",